tests/App/config/console.php

```diff
@@ -1,33 +1,23 @@
 <?php
 
+use Psr\Container\ContainerInterface;
+use Symfony\Component\Console\CommandLoader\ContainerCommandLoader;
+use Yiisoft\Yii\Console\Application;
+use Yiisoft\Yii\Queue\Tests\App\Benchmark\Controller;
+
 return [
-    'app' => [
-        'controllerMap' => [
-            'mysql-migrate' => [
-                '__class' => \Yiisoft\Yii\Console\Controllers\MigrateController::class,
-                'db' => 'mysql',
-                'migrationPath' => null,
-                'migrationNamespaces' => [
-                    'Yiisoft\Yii\Queue\Drivers\Db\migrations',
-                ],
-            ],
-            'sqlite-migrate' => [
-                '__class' => \Yiisoft\Yii\Console\Controllers\MigrateController::class,
-                'db' => 'sqlite',
-                'migrationPath' => null,
-                'migrationNamespaces' => [
-                    'Yiisoft\Yii\Queue\Drivers\Db\migrations',
-                ],
-            ],
-            'pgsql-migrate' => [
-                '__class' => \Yiisoft\Yii\Console\Controllers\MigrateController::class,
-                'db' => 'pgsql',
-                'migrationPath' => null,
-                'migrationNamespaces' => [
-                    'Yiisoft\Yii\Queue\Drivers\Db\migrations',
-                ],
-            ],
-            'benchmark' => \Yiisoft\Yii\Queue\Tests\App\Benchmark\Controller::class,
-        ],
-    ],
+    Application::class => static function (ContainerInterface $container) {
```

Comment:
That would likely mess with other console commands. Can we do it like here? https://github.com/yiisoft/yii-demo/blob/master/config/params.php#L27

Reply:
I didn't touch any test and its environment, except a couple of first commits and automatic refactoring. If you are ok with the basic archetecture I will adjust tests to it and to `yii-console`.